boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
This is not going to work on Windows.

Reply:
Updated to `#!/usr/bin/env bash`. Is this what you are asking for?

Reply:
> Updated to `#!/usr/bin/env bash`. Is this what you are asking for?

No, I am saying this shell script is not going to be able to run on Windows and the script should be written in, for example, Python.

Reply:
Understood. Let me see what I can do.